CMakeLists.txt

```diff
@@ -0,0 +1,179 @@
+# Copyright 2022 The Foundry Visionmongers Ltd
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.21)
+project(OpenAssetIOTest)
```

Comment:
[consider] Do we want to update this to match the repo name?

Reply:
Updated in dc370249

Reply:
[question] Should this affect the CMAKE options and such too?

Reply:
:thinking:

On the one hand, I could imagine one day having multiple `OpenAssetIO-Test-*` projects in a single repo, all testing different things, so we'd have to disambiguate on whatever `*` is.

But on the other hand, adding `CMAKE` to the end of the namespace could be confusing, since there are lots of built-in variables that are namespaced `CMAKE_`, so could give the impression we're overriding something built in.

I think I'd err on the side of keeping the namespace shorter.